engine/app/components/citizens_advice_components/asset_hyperlink_component.html.haml

```diff
@@ -0,0 +1,5 @@
+%a{ href: href }
+  = description
+  %span.cads-asset-type<
+    %span.cads-icon_file
+    \ #{number_to_human_size(size)}
```

Comment:
Because we're in a Rails context here we can use the `number_to_human_size` helper rather than the limited `" #{(asset_hyperlink['size'] / 1_000_000.0).round(1)} MB"` we are currently using.